sdk/nodejs/package.json

```diff
@@ -22,7 +22,7 @@
         "semver": "^6.1.0",
         "source-map-support": "^0.4.16",
         "ts-node": "^7.0.0",
-        "typescript": "^3.0.0",
+        "typescript": "=3.6.3",
```

Comment:
Nit - but can we use ~ to float on patch version but fix 3.6.x?

Totally fine with this as is though.